src/mercury_engine_data_structures/formats/bmsld.py

```diff
@@ -36,36 +37,37 @@
 
 Components = {
```

Comment:
is this even used? can it just be removed?

Reply:
It looks like all the possible component types but isn't used. I imagine it was going to be so each one could have it's own format for the arguments? Not sure.